docs/development/translations.rst

```diff
@@ -0,0 +1,300 @@
+Translations
+============
+
+Get Started
+-----------
```

Comment:
Maybe cross-reference background section here? ("Need more context? See .." or similar)

Reply:
Yes, indeed. Even just listing the first few questions:

- What is SecureDrop?
- Who uses SecureDrop?
- How is SecureDrop translated?

...could be a straightforward way to make clear that more background information is available, while only delaying the **Get Started** landmark minimally.

Reply:
Done.

Reply:
The bullet list format is unfortunately a bit confusing here from a navigational standpoint -- it could be seen as a table of contents, especially since it is so snugly positioned between a headline called "Get started" and a subheading called "Getting started with ..". Does it take me further into that section or somewhere else entirely? It's not at all obvious.

I would perhaps rename the "Get Started" section to "[Quick start guide](https://en.wikipedia.org/wiki/Quickstart_guide)", a standard term in technical documentation. Regardless, I would suggest replacing the list with a sentence, which reduces the likelihood that it will be interpreted as a navigational element for the entire document or the subsection it is anchored to. IMO we don't need to enumerate all three links, either -- just a single link for background on SecureDrop should be sufficient.

Reply:
Done. (I'm checking case conventions for section headers.)